src/Tokenizer/Tokens.php

```diff
@@ -892,11 +901,7 @@ public function insertSlices(array $slices): void
      */
     public function isChanged(): bool
     {
-        if ($this->changed) {
-            return true;
-        }
-
-        return false;
+        return $this->changed;
```

Comment:
i remember the idea here was to have different lines for different paths, so we can have explicit code coverage not just by one-liner covered, but to ensure both paths are covered for this crucial core-level functionality

@SpacePossum , should we move back to multiline approach, perhaps?

Reply:
I'm not sure I follow, what is the benefit of higher LOC's here?

Reply:
i remember initial reasoning. we wanted to ensure, initially, that we don't test only cases like "this->changed=true"

Reply:
oh like in the coverage it won't show if both `true` and `false` and checked.... that is true. I value a more direct and compact notation even if that means a bit less verbose coverage report.